guide/12-enrich-data-with-thematic-information/part6_country_enrich.ipynb

```diff
@@ -0,0 +1,1115 @@
+{
```

Comment:
<p>As in Part 1, facility 1 and facility 2 have higher populations and are diverse with lower incomes. Facility 3 is far younger, with larger households, less education, and lower incomes. Facility 4 and facility 5 are older, more educated, and have higher incomes. </p><p>If we are interested in opening a discount department store, facility 2 is the most attractive location, with facility 1 as a close second. The diversity and lower income can allow us to conclude that people will buy at lower prices. </p><p>If we are interested in opening a quick service restaurant, facility 3 may be the best option to meet the needs of a young, busy, and price conscious population. </p><p>Obviously, depending on the key characteristics of the business looking for a new location, the key demographic indicators will be different. Using geoenrichment and ArcGIS API for Python enables quick access to demographic variables for informed decision making.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>